### PR TITLE
Finalising `style.scss`, `overrides.scss`, and `settings.scss`

### PR DESCRIPTION
**Pull request by chris-pearce**

### CODE REVIEW

File: utilities/_u-widths.scss [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 /**
  * Apply percentage based width classes with the option to apply at all the
- * main breakpoints.
+ * main breakpoints. All the classes are the same as the Width settings found
+ * in: Core -> Settings -> Widths, so `$one-whole` would be `.one-whole`.
```
Comment (AlecRaeside) [inline utilities/_u-widths.scss:9]: its currently outputting as `.u-one-whole`

**Comment by AlecRaeside** [comment]
Not part of this PR, but the `bower_components/scally/` part of the import path should be a variable so If someone is not using bower they can change it in one place instead of four

**Comment by smgcode** [comment]
I agree with @AlecRaeside regarding the bower. otherwise LGTM.

**Comment by chris-pearce** [comment]
@AlecRaeside @smgcode you can't use a variable in an `@import` path.

**Comment by kllevin** [comment]
LGTM

**Pull request merged**